packages/cli/src/networks/dev.ts

```diff
@@ -1,21 +1,25 @@
 import {minimalChainConfig, mainnetChainConfig} from "@lodestar/config/presets";
 import {ACTIVE_PRESET, PresetName} from "@lodestar/params";
+import {NetworkData} from "./index.js";
 
-let chainConfig;
-switch (ACTIVE_PRESET) {
-  case PresetName.mainnet:
-    chainConfig = mainnetChainConfig;
-    break;
-  case PresetName.minimal:
-    chainConfig = minimalChainConfig;
-    break;
```

Comment:
Instead of adding a network specific `getNetworkData`, I think we can just add another `case` for `PresetName.gnosis`. (And perhaps remove the `default` case)

Reply:
(We probably want to support any preset for the dev command anyways)

Reply:
> I think we can just add another case for PresetName.gnosis

lets do this for now to get this fix out asap, I still think it is not a good that we execute code related to `dev` command if we execute other commands such as `beacon`

Reply:
Agree that its not great to be running this code, just looking for the smaller diff for quick fix.

Also looked into turning the network imports into an `await import` but that also is a big change since getNetworkData and some downstream consumers are all sync functions

Reply:
> Also looked into turning the network imports into an await import

yeah that was also the initial solution I wanted to use until I figured that the downstream call chain just never ends and had like 30+ changes, I still have most of those changes stashed if we want to apply this solution later